packages/next-swc/package.json

```diff
@@ -12,7 +12,9 @@
     "build-wasm": "wasm-pack build crates/wasm --scope=next",
     "cache-build-native": "echo $(ls native)",
     "rust-check": "cd ../../; cargo fmt -- --check && cargo clippy --all -- -D warnings -A deprecated && cargo check -p next-dev --no-default-features --features cli,custom_allocator,rustls-tls,__internal_nextjs_integration_test && rm -rf target",
-    "test-cargo-unit": "cargo test --workspace --exclude next-dev-tests"
+    "test-cargo-unit": "cargo test --workspace --exclude next-dev-tests",
+    "test-cargo-integration": "cross-env CARGO_PROFILE_RELEASE_LTO=false cargo nextest run -p next-dev-tests --release --features tracing/release_max_level_off --no-fail-fast",
```

Comment:
I feel like we should just remove LTO from the cargo.toml and only use it for the actual release builds